src/Graphics/AnimationFlashing.cpp

```diff
@@ -1,13 +1,21 @@
 #include "AnimationFlashing.h"
 
-void AnimationFlashing::run(int64_t millis) {
-	if (millis > _lastTimeMillis + _interval) {
+void AnimationFlashing::run(int64_t millis)
+{
+	if (millis > _lastTimeMillis + _interval)
+	{
 		_lastTimeMillis = millis;
 
 		if (_isDrawn)
+		{
 			_shape->clear();
+			_blink->set(0);
```

Comment:
Every time an animation blinks, it will change the shared property. The animationflashing class is currently used for hazards and turn signals, so this may need some code in main to prevent the hazards button led from flashing when the hazard button is not pressed.